src/test/java/com/hocs/test/pages/dcu/DCUProgressCase.java

```diff
@@ -264,10 +267,19 @@ public void generateDCUSearchCaseData(String infoValue, String infoType) throws
                 createCase.createCSCaseOfType(infoValue);
                 break;
             case "RECEIVED ON OR AFTER DATE":
-                createCase.createCaseReceivedFiveDaysBeforeOrAfterDate("MIN", "After", infoValue);
-                break;
             case "RECEIVED ON OR BEFORE DATE":
-                createCase.createCaseReceivedFiveDaysBeforeOrAfterDate("MIN", "Before", infoValue);
+                dashboard.selectCreateSingleCaseLinkFromMenuBar();
+                if (!nextButton.isVisible()) {
+                    dashboard.selectCreateSingleCaseLinkFromMenuBar();
+                }
+                createCase.selectCaseType("MIN");
+                clickTheButton("Next");
+                createCase.editReceivedDate(infoValue);
+                createCase.storeCorrespondenceReceivedDate();
+                documents.uploadFileOfType("docx");
+                clickTheButton("Create case");
+                confirmationScreens.storeCaseReference();
+                dashboard.goToDashboard();
```

Comment:
This appears to be achieving the same as calling createCase.createCSCase("MIN", false, infoValue), which would be a lot tidier?